docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -350,6 +387,10 @@ spec:
         replicas: 1
 EOF
 ----
+
+* Replace <address_pool> with the name of your network definition.
+* Replace <loadBalancer_IP> with the LoadBalancer IP address.
+* Replace <networkAttachment_name> with the name of your network.
```

Comment:
I just noticed this bit also should be conditional for OSPDO, as in the non-OSPDO docs we provide example values instead of the placeholders.

Reply:
ok